app/src/main/java/fr/free/nrw/commons/utils/LangCodeUtils.java

```diff
@@ -20,4 +25,12 @@ public static String fixLanguageCode(String code) {
             return code;
         }
     }
+    public static Resources getLocalizedResources(Context context, Locale desiredLocale) {
```

Comment:
Every created method must have Javadoc. Just say what service the method offers.